<launch>
  <!-- Simulationszeit aktivieren -->
  <param name="use_sim_time" value="true"/>

  <!-- AprilTag detector -->
  <include file="$(find landmark_mapper)/launch/apriltag.launch" />

  <!-- Color Sampler node -->
  <node name="color_sampler_node" pkg="landmark_mapper" type="color_sampler_node" output="screen">
    <param name="output_path" value="$(find landmark_mapper)/output/landmarks_raw.yaml"/>
  </node>

  <!-- Landmark Mapper node -->
  <node name="landmark_mapper_node" pkg="landmark_mapper" type="landmark_mapper_node" output="screen">
    <param name="output_path" value="$(find landmark_mapper)/output/landmarks_mapped.yaml"/>
  </node>

  <!-- Debug -->
  <node pkg="rqt_image_view" type="rqt_image_view" name="image_debug" output="screen"
        args="/camera/image"/>
</launch>
